get-offline.js

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Where is this file called?

Reply:
@tlfrd It's set as `bin.offline` in package.json so users can either install `@financial-times/d3-bootloader` globally or run `npx @financial-times/d3-bootloader` and it will:

1. Download all dependencies for offline use to the local directory
2. Rewrite the d3-bootloader script tag to use the offline version.